api/modules/users/views.py

```diff
@@ -206,3 +206,20 @@ def remove_profile_image(request):
     user.profile.profile_image = None
     user.save()
     return Response("Profile image succesfully removed.", status=status.HTTP_200_OK)
+
+
+@api_view(['GET'])
+def remove_user_status(request):
+    """
+    Remove user status of a user
+    :param request:
+    :return: 200 successful
+    """
+    try:
+        user = request.user.profile
```

Comment:
What if the user has no profile?

Reply:
User will always have a profile because of the post save signal that we have used...